documentation/src/docs/getting-started.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Getting Started with Spark
+sidebar_label: Getting Started
+---
+
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+
+import TOCInline from '@theme/TOCInline';
+import { IfHaveFeature, IfMissingFeature } from 'nautilus-docs';
+
+<TOCInline toc={toc} />
+
+You can run Apache Spark applications written in Java, Scala, or Python. Using Python is the easiest to get started.
+
+## Prepare Development Environment
+
+<IfHaveFeature feature="nautilus">
+
+:::tip Streaming Data Platform
+SDP users can optionally skip this entire *Prepare Development Environment* section. However, it is often useful to develop applications locally in a sandbox environment before deploying to a production system such as SDP. For this reason, we recommend that this section is not skipped.
+:::
+
+</IfHaveFeature>
+
+### Install Operating System
+
+Install Ubuntu 20.04 LTS. Other operating systems can also be used but the commands below have only been tested
+on this version.
+
+### Install Java 11
+
+```shell
+sudo apt-get install openjdk-11-jdk
+```
+
+You may have multiple versions of Java installed. Ensure that Java 11 is the default with the command below.
+
+```shell
+sudo update-alternatives --config java
+```
+
+### Run Pravega
+
+This will run a development instance of Pravega locally. The transaction parameters allow transactions to remain open for up to 30 days without lease renewals.
+
+```shell
+cd
+git clone https://github.com/pravega/pravega
+cd pravega
+git checkout r0.9
+./gradlew startStandalone \
+    -Dcontroller.transaction.lease.count.max=2592000000 \
+    -Dcontroller.transaction.execution.timeBound.days=30
+```
+
+### Install Apache Spark
+
+This will install a development instance of Spark locally.
+
+Download https://www.apache.org/dyn/closer.lua/spark/spark-3.0.2/spark-3.0.2-bin-hadoop2.7.tgz.
+
+```shell
+mkdir -p ~/spark
+cd ~/spark
+tar -xzvf ~/Downloads/spark-3.0.2-bin-hadoop2.7.tgz
+ln -s spark-3.0.2-bin-hadoop2.7 current
+export PATH="$HOME/spark/current/bin:$PATH"
+```
+
+By default, the script `run_spark_ap.sh` will use an in-process Spark mini-cluster
+that is started with the Spark job (`--master local[2]`).
+
```

Comment:
It is better if we add an additional section here about how to setup from an empty project instead of directly jump into an exsiting one.
Things like "choose a correct Spark connector artifact depending on your Spark and Pravega version" and "How to import the connector dependency(java/python) and package the application into jar files(java)" is not mentioned in the whole doc here. I think this is useful for starting

Reply:
With the removal of `run_spark_app.sh`, the only thing needed for a new Python Spark app is the new .py file. The documentation currently does not cover how to write a Java Spark app.